src/tests/test_sysargv.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import sys
+
+
+class SysArgvTests(unittest.TestCase):
+    """Test sys.argv state."""
+    def testSysArgvState(self):
+        """Test sys.argv state."""
+        argv = list(sys.argv) # need to make a copy of list
+        import clr
+        self.assertTrue(argv == sys.argv)
```

Comment:
this doesn't make sense. Did importing `clr` previous cause `sys.argv` to change? Other tests `import clr` before this test, that may change being too.